src/js/index.js

```diff
@@ -0,0 +1,218 @@
+import CreatePage from './components/createHtml/createHtml';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const createPage = new CreatePage();
+
+const LANG = { RU: 'ru', EN: 'en' };
+
+const state = {
+  lang: LANG.RU,
+  capsLock: false,
+};
+
+const KEYBOARD = {
+  tab: '  ',
+  space: ' ',
+  enter: '\n',
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
+    state.lang = language;
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  createPage.create();
+  board.createKeyBoard(state.lang);
+
+  const keyBoard = document.querySelector('.keyboard-list');
+
+  document.addEventListener('keydown', (event) => {
+    const switcherCtrlAlt = event.ctrlKey && event.altKey;
+    const switcherShiftAlt = event.shiftKey && event.altKey;
+    if (switcherCtrlAlt || switcherShiftAlt) {
+      state.lang = state.lang === LANG.RU ? LANG.EN : LANG.RU;
+      keyBoard.innerHTML = '';
+      keyBoard.innerHTML = board.generateButtons(state.lang);
+    }
+  });
+
+  const textarea = document.querySelector('.textarea');
+  const capsLock = document.querySelector('.capslock');
+
+  const capsLockHandler = () => {
```

Comment:
If the file is larger than 100 lines, you should consider splitting it. Otherwise it's hard to read. You can move out all your handlers into a separate file or at least move them out outside the listener. If a listener needs some DOM elements to work with (like buttons and capsLock in your case), you can pass them as listener arguments.

Reply:
done